### PR TITLE
US661466   doc update dap secure connection support

### PR DESCRIPTION
**Pull request by DStatWriter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,72 @@ When upgrading to this version of Debugger for Mainframe from an older version,
...
+                "interTestPort": 12345,
+                
+                "interTestUserName": "PMFID",
+                
+                "interTestSecure": true,
+                
+                "cicsApplId": "CICSappID"
+            }
+        
+Note: **interTestSecure": true** will display if Secure connection has been succesfully configured.
```
Comment (zeibura) [inline README.md:90]: This step is a bit confusing. Shouldn't it be:
4. Check launch.json looks like this / has "interTestSecure: true"
5. Run a test debug session

File: README.md [hunk 2]
```diff
@@ -28,6 +28,72 @@ When upgrading to this version of Debugger for Mainframe from an older version,
...
+- InterTest server running and configured to support secure communication.
+- Security certificate for the InterTest server.
+
+**Follow these steps**
+
+1. Download the Server Certificate to your local machine
+
+2. Import certificate with entry name HOSTNAME to the trust store, for example:
+
+    *jdk0.0.0_000\abc\Lib\security* named cacerts of the JRE under which the Debugger for Mainframe DA client is running.
```
Comment (zeibura) [inline README.md:43]: This step is grammatically incorrect, and not very clearly written. What does "named cacerts" mean on the second line? What is that path an example of and is it really necessary?
Can't we just say "import the certificate to the trust store of the JRE under which the Debugger for Mainframe DA client is running. Use the entry name HOSTNAME."

File: README.md [hunk 3]
```diff
@@ -28,6 +28,72 @@ When upgrading to this version of Debugger for Mainframe from an older version,
...
+
+3. Proceed using Command Line, or a UI Tool:
```
Comment (zeibura) [inline README.md:45]: Is this actually a new step or is it just instructions on how to do what is supposedly described in step 2? Both seem to be about importing the certificate.

**Comment by DStatWriter** [comment]
Closed and moved

**Pull request closed without merging**